infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -142,9 +142,13 @@ protected void processRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRo
      */
     @ServerHandler(type = CorfuMsgType.TAIL_REQUEST)
     public void handleTailRequest(CorfuPayloadMsg<TailsRequest> msg, ChannelHandlerContext ctx, IServerRouter r) {
-        log.debug("handleTailRequest: received a tail request {}", msg);
+        log.trace("handleTailRequest: received a tail request {}, type: {}", msg.getPayload().getReqType());
         batchWriter.<TailsResponse>addTask(TAILS_QUERY, msg)
-                .thenAccept(tailsResp -> r.sendResponse(ctx, msg, CorfuMsgType.TAIL_RESPONSE.payloadMsg(tailsResp)))
+                .thenAccept(tailsResp -> {
+                    log.debug("handleTailRequest: global tail: {}, number of streams: {}",
```

Comment:
So, essentially we have the same number of log messages flooding the log. I don't see any improvements here.

Reply:
The perpose of this patch is not to reduce logs, which you already did in a earlier patch. This is what we discussed earlier to replace initial logs with more meaning messages which include the query results.